bin/git-ignore-io

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+gitignore_io_url="https://www.gitignore.io/api/"
+
+default_path="$HOME/.gi_list"
+gi_list=`cat $default_path | tr "," "\n\n"`
+
+update_gi_list() {
+    curl -L -s  "${gitignore_io_url}/list" > ~/.gi_list
+}
+
+print_in_alphabetical_order() {
+    for i in {a..z};
+    do
+        echo "$gi_list" | grep "^$i" | tr "\n" " "
+        echo
+    done
+}
+
+print_in_table_format() {
+    table_width=5
+    echo "$gi_list" | xargs -n $table_width | column -t
+}
+
+search() {
+    for type in $gi_list;
+    do
+        if [[ $type == *$1* ]]
+        then
+            echo $type
+        fi
+    done
+}
+
+print_last_modified_time() {
+    local gi_list_date=`stat -f "%t%Sm" $default_path`
```

Comment:
Just tested this command on my machine(Linux).
Unfortunately, the `-f` means file system on GNU coreutils: http://linux.die.net/man/1/stat .
We have to find another way to do this thing.
Please GNU and BSD at the same time, is one of the difficulty to write a portable shell script.
Keep it up!